smithy-swift-codegen/src/test/kotlin/ServiceRenamesTests.kt

```diff
@@ -144,13 +144,6 @@ extension RestJsonProtocolClientTypes.RenamedGreeting {
         guard let value else { return }
         try writer["salutation"].write(value.salutation)
     }
```

Comment:
This and following are test files, the `NeedsReaderWriterTransformer` has to be applied to the text contexts, and some test expectations no longer have both a reader & writer.